Assets/switchCamera.cs

```diff
@@ -1,18 +1,19 @@
 ﻿using UnityEngine;
 using System.Collections;
 
-public class switchCamera : MonoBehaviour {
+public class SwitchCamera : MonoBehaviour
```

Comment:
The file name should start with uppercase too. Donno why lowercase here. Works in my local directory.